docs/guides/get-started-with-primitives.mdx

```diff
@@ -264,31 +264,79 @@ print(f"  > Metadata: {result.metadata[0]}")
+Unlike provider-specific primitives, backend primitives are generic implementations that can be used with an arbitrary 
+`backend` object, as long as it implements the [`Backend`](/api/qiskit/qiskit.providers.Backend) interface.
+
+The `Sampler` primitive can be run with any provider by using [`qiskit.primitives.BackendSamplerV2`](/api/qiskit/qiskit.primitives.BackendSamplerV2)
+or the legacy V1 implementation, [`qiskit.primitives.BackendSampler`](/api/qiskit/qiskit.primitives.BackendSampler). 
+Likewise, the `Estimator` primitive can be run with any provider using [`qiskit.primitives.BackendEstimatorV2`](../api/qiskit/qiskit.primitives.BackendEstimatorV2), 
+or the legacy V1 implementation, [`qiskit.primitives.BackendEstimator`](/api/qiskit/qiskit.primitives.BackendEstimator).
 
 Some providers implement primitives natively (see [the Qiskit Ecosystem page](https://qiskit.github.io/ecosystem#provider) for more details).
 
 ### Example: BackendEstimator
 
+<Tabs>
+  <TabItem value="BackendEstimatorV2" label="BackendEstimator V2">
 ```python
+from qiskit.primitives import BackendEstimatorV2
+from <some_qiskit_provider> import QiskitProvider
+
+provider = QiskitProvider()
+backend = provider.get_backend('backend_name')
+estimator = BackendEstimatorV2(backend)
+```
+  </TabItem>
+
+  <TabItem value="BackendEstimatorV1" label="BackendEstimator (V1)">
+  ```python
 from qiskit.primitives import BackendEstimator
 from <some_qiskit_provider> import QiskitProvider
 
 provider = QiskitProvider()
 backend = provider.get_backend('backend_name')
 estimator = BackendEstimator(backend)
 ```
+  </TabItem>
 
 ### Example: BackendSampler
 
+<Tabs>
+  <TabItem value="BackendSamplerV2" label="BackendSampler V2">
 ```python
+from qiskit.primitives import BackendSamplerV2
+from <some_qiskit_provider> import QiskitProvider
+
+provider = QiskitProvider()
+backend = provider.get_backend('backend_name')
+sampler = BackendSamplerV2(backend)
+```
+  </TabItem>
+
+  <TabItem value="BackendSamplerV1" label="BackendSampler (V1)">
+  ```python
 from qiskit.primitives import BackendSampler
 from <some_qiskit_provider> import QiskitProvider
 
 provider = QiskitProvider()
 backend = provider.get_backend('backend_name')
 sampler = BackendSampler(backend)
 ```
+  </TabItem>
+
+### Similarities and differences between backend and Runtime primitives (V2)
+
+- The inputs to [`qiskit.primitives.BackendSamplerV2`](/api/qiskit/qiskit.primitives.BackendSamplerV2) and [`qiskit.primitives.BackendEstimatorV2`](../api/qiskit/qiskit.primitives.BackendEstimatorV2)
+follow the same PUB format as the primitives in Qiskit Runtime, so do the outputs, see (link to https://docs.quantum.ibm.com/guides/primitive-input-output). 
+However, there can be differences in the fields of the returned metadata.
```

Comment:
I decided not to go into further detail because the metadata isn't a fixed part of the interface and the content has been changing quite a lot, so listing specific fields I think increases the maintenance load of the page.

Reply:
Seems reasonable.